### PR TITLE
load organisation to redux for token in create user

### PR DESCRIPTION
**Pull request by wangfowen**

### CODE REVIEW

File: app/client/api/organisationApi.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {generateFormattedURL, getToken, handleResponse} from "../utils";
```
Comment (wangfowen) [inline app/client/api/organisationApi.js]: more code repeat to refactor out i'm sorryyyy

File: app/client/components/user/CreateUserForm.tsx [hunk 2]
```diff
@@ -132,6 +135,8 @@ export default connect(state => {
   const selector = formValueSelector('createUser');
   return {
     businessUsageValue: selector(state, 'businessUsage'),
+    // @ts-ignore
+    organisation: state.organisation.data
```
Comment (wangfowen) [inline app/client/components/user/CreateUserForm.tsx]: i thiiink we need to convert all the reducers to typescript to get the typing working correctly for this, would be really nice if we do

File: app/client/sagas/organisationSagas.js [hunk 3]
```diff
@@ -1 +1,48 @@
-//TODO: use the new organisation reducers
```
Comment (wangfowen) [inline app/client/sagas/organisationSagas.js]: again more repeat :(

File: app/server/me_api/organisation.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+from server.schemas import me_organisation_schema
+from server.utils.auth import requires_auth
+
+
+class MeOrganisationAPI(MethodView):
+    @requires_auth
+    def get(self):
+
+        organisation = g.user.default_organisation
```
Comment (wangfowen) [inline app/server/me_api/organisation.py]: do we want to use default organisation?

File: app/client/api/organisationApi.js (hunk 1 above)
Comment (enjeyw) [reply app/client/api/organisationApi.js]: lolllll my fault for building it this way in the first place

File: app/client/components/user/CreateUserForm.tsx (hunk 2 above)
Comment (enjeyw) [reply app/client/components/user/CreateUserForm.tsx]: yeah, it's annoying that it can't be incremental

Comment (enjeyw) [reply app/client/components/user/CreateUserForm.tsx]: would be good to do at the same time as eliminating that YUGGGEEE amount of copypasta boilerplate

File: app/server/me_api/organisation.py (hunk 4 above)
Comment (enjeyw) [reply app/server/me_api/organisation.py]: shouldn't this take the active organisation defined by the query string?

Comment (wangfowen) [reply app/server/me_api/organisation.py]: so every request now passes in org in the query string? always as 'org'?

**Pull request merged**